crates/uv/src/commands/reporters.rs

```diff
@@ -55,6 +55,21 @@ struct BarState {
     size: FxHashMap<usize, Option<u64>>,
     /// A monotonic counter for bar IDs.
     id: usize,
+    /// The maximum length of all bar names encountered.
+    max_len: usize,
+}
+
+impl Default for BarState {
+    fn default() -> Self {
+        Self {
+            headers: usize::default(),
+            sizes: Vec::<_>::default(),
+            bars: std::collections::HashMap::<_, _, _>::default(),
+            size: std::collections::HashMap::<_, _, _>::default(),
+            id: usize::default(),
```

Comment:
You should be able to clean this up a little bit.
```suggestion
            headers: 0,
            sizes: Vec::default(),
            bars: FxHashMap::default(),
            size: FxHashMap::default(),
            id: 0,
```